doc/cluster-overview.md

```diff
@@ -0,0 +1,15 @@
+# Cluster Overview:
+
+Pravega is a storage system for data streams that has an innovative design and an attractive set of features to cope with today’s Stream processing requirements (e.g., event ordering, scalability, performance, etc.). For deploying Pravega, we have to install the following components as a pre-requisite. Pravega can be deployed in different environments like OpenShift, KubeSpray, GKE, EKS etc
```

Comment:
Can we mention the services of Pravega and their properties? Statelessness/statefulness, number of allowed/intended instances, ports, light overview on theory of operation, which Pravega processes connect to the services below, etc.?